cmd/meroxa/root/functions/logs.go

```diff
@@ -0,0 +1,80 @@
+package functions
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/meroxa/cli/cmd/meroxa/builder"
+	"github.com/meroxa/cli/log"
+	"github.com/meroxa/meroxa-go/pkg/meroxa"
+)
+
+var (
+	_ builder.CommandWithDocs    = (*Logs)(nil)
+	_ builder.CommandWithArgs    = (*Logs)(nil)
+	_ builder.CommandWithClient  = (*Logs)(nil)
+	_ builder.CommandWithLogger  = (*Logs)(nil)
+	_ builder.CommandWithExecute = (*Logs)(nil)
+)
+
+type functionLogsClient interface {
+	GetFunctionLogs(ctx context.Context, nameOrID string) (*http.Response, error)
+}
+
+type Logs struct {
+	client functionLogsClient
+	logger log.Logger
+
+	args struct {
+		NameOrID string
```

Comment:
Same as meroxa-go, it's best to refer to it as UUID
```suggestion
		NameOrUUID string
```